src/ucs/datastruct/array.h

```diff
@@ -314,14 +308,14 @@ ucs_array_old_buffer_set_null(void **old_buffer_p)
  * @return Current capacity of the array
  */
 #define ucs_array_capacity(_array) \
-    ((_array)->capacity & UCS_ARRAY_CAP_MASK)
+    ((_array)->capacity)
```

Comment:
minor: can be on the same line (it is short now)

Reply:
Sure, it can, but then it will be different style from other similar macros declarations.
I would prefer to keep the same style across this header file